suite-common/redux-utils/package.json

```diff
@@ -19,6 +19,7 @@
         "@suite-common/wallet-config": "workspace:*",
         "@suite-common/wallet-types": "workspace:*",
         "@trezor/connect": "workspace:*",
+        "lodash": "^4.17.21",
```

Comment:
is it worth introducing a new library?

Reply:
It's already used in some places, also if you check import it's possible to import only one function from lodash